core/templates/pages/blog-admin-page/navbar/blog-admin-navbar.component.html

```diff
@@ -0,0 +1,137 @@
+
+<div class="blog-admin-navbar">
```

Comment:
The structure of the navbar is same as release coordinator navbar file( or admin page navbar file)